src/mozilla_sec_eia/ex_21/extractor.py

```diff
@@ -0,0 +1,193 @@
+"""Fine-tune LayoutLM to extract Ex. 21 tables.
```

Comment:
I changed this from a class to a collection of functions because it worked a bit better with mlflow that way. The layout and interface of this module can definitely be refined/changed as we start to actually use the model more.